app/src/main/java/com/example/findissues/ui/fragments/HomeFragment.kt

```diff
@@ -85,39 +98,51 @@ class HomeFragment : Fragment() {
             pinnedRepoAdapter.setUpPinnedRepoList(it)
         }
 
-        with(binding) {
-            tvFollowers.setOnClickListener {
-                setUpFragment(FollowersFragment())
-            }
-            tvFollowing.setOnClickListener {
-                setUpFragment(FollowingFragment())
-            }
-            tvRepositories.setOnClickListener {
-                setUpFragment(RepositoryFragment())
-            }
-            tvStarred.setOnClickListener {
-                setUpFragment(StarredFragment())
+//        with(binding) {
+//            tvFollowers.setOnClickListener {
+//                setUpFragment(FollowersFragment())
+//            }
+//            tvFollowing.setOnClickListener {
+//                setUpFragment(FollowingFragment())
+//            }
+//            tvRepositories.setOnClickListener {
+//                setUpFragment(RepositoryFragment())
+//            }
+//            tvStarred.setOnClickListener {
+//                setUpFragment(StarredFragment())
+//            }
+//        }
+
+//        return binding.root
+        return ComposeView(requireContext()).apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.Default)
+            setContent {
+                HomeScreen(
+                    user = user,
+                    onFollowersClick = { setUpFragment(FollowersFragment()) },
+                    onFollowingClick = { setUpFragment(FollowingFragment()) },
+                    onRepositoriesClick = { setUpFragment(RepositoryFragment()) },
+                    onStarredClick = { setUpFragment(StarredFragment()) },
+                    goToTwitter = { Browser(requireContext()).launch(goToTwitter(it)) }
```

Comment:
create a `theme.kt` file and define AppTheme in that file. Then wrap HomeScreen inside AppTheme. It will look like:
```
AppTheme {
         HomeScreen(
              user = user,
              onFollowersClick = { setUpFragment(FollowersFragment()) },
              onFollowingClick = { setUpFragment(FollowingFragment()) },
              onRepositoriesClick = { setUpFragment(RepositoryFragment()) },
              onStarredClick = { setUpFragment(StarredFragment()) },
              goToTwitter = { Browser(requireContext()).launch(goToTwitter(it)) 
          }
     }